sunpy/net/helio/hec.py

```diff
@@ -149,7 +148,7 @@ def search(self, *args, **kwargs):
         table = qrdict.get('table_name', None)
         if table:
             if isinstance(table, bytes):
-                warn('type `bytes` for table_name is deprecated, use `str` instead.', SunpyDeprecationWarning)
+                warn_deprecated('type `bytes` for table_name is deprecated, use `str` instead.')
```

Comment:
Should this be removed instead? I forgot what release this was added into.

Reply:
🤷 I don't know enough about the warning to change it.